### ISSUE TITLE
Make listBibl mandatory in sourceDesc

### ISSUE THREAD

**Issue by ttasovac** [DARIAH-ERIC/lexicalresources#188]
This is a placeholder from the Lexical Resoruces Summit 2023.

If there is a `sourceDesc`, we should have at least one `listBibl` mandatory in it. `@type` suggested values: `dictionaries`, `literature`, `corpora`.

**Comment by ttasovac** [comment]
Schema to test with: https://raw.githubusercontent.com/DARIAH-ERIC/lexicalresources/feature/listBibl/Schemas/TEILex0/out/TEILex0.rng

@anacastrosalgado when you have a sec, could you please post here the `sourceDesc` from Morais, which should now contain two `listBibl` elements, one of the type `dictionaries` which should have the original print dictionaries in `biblStruct`, and then `listBibl` of the type `literature` which should have 2-3 encoded examples from Morais' sources.

**Comment by anacastrosalgado** [comment]
Morais `sourceDesc`:

```xml
<sourceDesc>
   <listBibl type="dictionaries">
      <biblStruct type="book" xml:id="Morais1789">
         <monogr>
            <title level="m" type="main">Diccionario da lingua portugueza composto pelo padre
               D. Rafael Bluteau, reformado, e accrescentado por Antonio de Moraes Silva,
               natural do Rio de Janeiro</title>
            <title level="m" type="sub">A – K</title>
            <author>
               <persName>
                  <forename>António de</forename>
                  <surname>Morais Silva</surname>
               </persName>
            </author>
            <imprint>
               <pubPlace>Lisboa</pubPlace>
               <publisher>Officina de Simão Thaddeo Ferreira</publisher>
               <pubPlace>Lisboa</pubPlace>
               <date when="1789">1789</date>
               <note>Com Licença da Real Meza da Comissão Geral, sobre o Exame, e Censura dos
                  Livros.</note>
               <!-- This sentence describes the store where the dictionary was for sale. We decided to use note (04.02.2023) -->
               <note>Vende-ſe na loja de Borel Borel, e Companhia, quaſi defronte da Igreja
                  nova de Noſſa Senhora dos Martyres, na eſquina.</note>
            </imprint>
            <extent>Tomo primeiro</extent>
            <extent>752 pp.</extent>
         </monogr>
      </biblStruct>
      <biblStruct>
         <monogr>
            <title level="m" type="main">Diccionario da lingua portugueza composto pelo padre
               D. Rafael Bluteau, reformado, e accrescentado por Antonio de Moraes Silva,
               natural do Rio de Janeiro</title>
            <title level="m" type="sub">L – Z</title>
            <!-- SEE corresp -->
            <author corresp="https://isni.org/isni/0000000083438040">
               <persName>
                  <forename>António de</forename>
                  <surname>Morais Silva</surname>
               </persName>
            </author>
            <imprint>
               <publisher>Officina de Simão Thaddeo Ferreira</publisher>
               <pubPlace>Lisboa</pubPlace>
               <date>1789</date>
            </imprint>
            <extent>Tomo segundo</extent>
            <extent>541 pp.</extent>
         </monogr>
      </biblStruct>
   </listBibl>
   <listBibl type="literature">
      <biblStruct resp="#MorDigital">
         <monogr corresp="https://purl.pt/29333">
            <title>Abecedario Real e Regia Instrucçam dos Principes Lusitanos, composto
               de 63. discursos Politicos, &amp; Moraes : offerecido ao Serenissimo
               Principe Dom Joam N.S. / pelo M.R.P. Fr. Joam dos Prazeres, Prègador
               Gèral, &amp; Chronista mòr da Religiaõ do Principe dos Patriarcas Sam
               Bento</title>
            <author>
               <persName>
                  <surname>Prazeres</surname>
                  <forename>João dos</forename>
               </persName>
            </author>
            <imprint>
               <date>1692</date>
               <pubPlace>Lisboa</pubPlace>
               <publisher>na Officina de Miguel Deslandes, Impressor de S.
                  Magestade</publisher>
               <!-- See link to BND -->
               <note>More information found in BND ; 191 p.</note>
            </imprint>
         </monogr>
      </biblStruct>
      <biblStruct xml:id="Academ.sing">
         <monogr corresp="#bibl. https://purl.pt/21936">
            <title>Academia dos ſingulares de Lisboa dedicadas a Apollo</title>
            <author>
               <persName>
                  <surname>Faria</surname>
                  <forename>André Leitão de</forename>
               </persName>
            </author>
            <imprint>
               <date>1665</date>
               <pubPlace>Lisboa</pubPlace>
               <publisher>na Officina de Henrique Valente de Oliveira</publisher>
               <biblScope unit="volume">2 t. em 2 vol.</biblScope>
               <note>More information found in BND; 2 vol.</note>
            </imprint>
         </monogr>
      </biblStruct>
      <!-- [...] -->
   </listBibl>
</sourceDesc>
```

**Comment by ttasovac** [comment]
merged into dev with #195
